ocaml/xapi/xapi_pif.ml

```diff
@@ -26,16 +26,12 @@ open Threadext
 open Network
 
 let get_device_pci ~__context ~host ~device =
-  let open Network_stats in
-  let stats = read_stats () in
-  try
-    let stat = List.assoc device stats in
-    let expr = Db_filter_types.(And (Eq (Field "pci_id", Literal (stat.pci_bus_path)),
-                                     Eq (Field "host", Literal (Ref.string_of host)))) in
-    match Db.PCI.get_refs_where ~__context ~expr with
-    | [pci] -> pci
-    | _  -> Ref.null
-  with Not_found -> Ref.null
+  let pci_bus_path = Net.Interface.get_pci_bus_path "get_pci_bus_path" ~name:device in
```

Comment:
Use `let dbg = Context.string_of_task __context in` for the debug-info in this call (see below in `refresh_internal`.

Reply:
sure